src/components/calcite-select/calcite-select.tsx

```diff
@@ -0,0 +1,303 @@
+import {
+  Component,
+  Host,
+  h,
+  Element,
+  Prop,
+  VNode,
+  Listen,
+  Method,
+  EventEmitter,
+  Event
+} from "@stencil/core";
+import { focusElement, getElementDir } from "../../utils/dom";
+import { Scale, Theme } from "../../interfaces/common";
+import { CSS } from "./resources";
+import { CSS_UTILITY } from "../../utils/resources";
+
+type OptionOrGroup = HTMLCalciteOptionElement | HTMLCalciteOptionGroupElement;
+type NativeOptionOrGroup = HTMLOptionElement | HTMLOptGroupElement;
+
+function isOption(optionOrGroup: OptionOrGroup): optionOrGroup is HTMLCalciteOptionElement {
+  return optionOrGroup.tagName === "CALCITE-OPTION";
+}
+
+function isOptionGroup(
+  optionOrGroup: OptionOrGroup
+): optionOrGroup is HTMLCalciteOptionGroupElement {
+  return optionOrGroup.tagName === "CALCITE-OPTION-GROUP";
+}
+
+@Component({
+  tag: "calcite-select",
+  styleUrl: "calcite-select.scss",
+  shadow: true
```

Comment:
How does this respond inside a `<form>`?  Can the form gain access to the data in the select on submit?

Reply:
UPDATE: sounds like we can add an enhancement ticket to add this support as it is not a requirement for v1.